From: Martin Pitt <mpitt@debian.org>
Date: Mon, 14 Nov 2016 22:41:23 +0100
Subject: Use python-config instead of sysconfig.get_python_inc()

The latter does not work with multiarch-split include files.

Bug-Ubuntu: https://launchpad.net/bugs/1094246
---
 setup/extensions.py | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/setup/extensions.py b/setup/extensions.py
index d309f5f..6a6d6f2 100644
--- a/setup/extensions.py
+++ b/setup/extensions.py
@@ -335,7 +335,11 @@ if isunix:
 if islinux:
     cflags.append('-pthread')
     ldflags.append('-shared')
-    cflags.append('-I'+sysconfig.get_python_inc())
+    import sys
+    config = subprocess.Popen([sys.executable + '-config', '--includes'],
+                              stdout=subprocess.PIPE)
+    cflags += config.communicate()[0].strip().split()
+    assert config.returncode == 0
     ldflags.append('-lpython'+sysconfig.get_python_version())
 
 
